docs/readme-cmake.md

```diff
@@ -65,6 +65,8 @@ Additionally, if you want to build unit tests:
 ```
 catch2
 ```
+Since Catch2 version 3.x is needed (3.4 works), you will have to install it with something like `vcpkg`, rather than apt.
```

Comment:
I don't think this makes sense without the context of the discussion in this PR; as far as I am aware, of the supported distro versions above, it is only Ubuntu 22.04 that doesn't provide an adequate version of catch.
We should also be clearer about what version is required: "3.x" means anything from 3.0 onwards. It seems strange to then say "3.4" works; do 3.0 through 3.3 not work? If so, then we should just say 3.4 or newer is required. If 3.0 works, then we've already covered that by saying "3.x".
```suggestion
While sufficient versions of other dependencies are available, Ubuntu 22.04 does not provide an up to date version of catch2 (3.0 or newer is required), so this will need to be built from source if unit tests are desired.
```